contentScript.js

```diff
@@ -124,8 +124,11 @@ if (window.location.href.indexOf("firstthings.com") !== -1) {
 
 if (window.location.href.indexOf("bloomberg.com") !== -1) {
     document.addEventListener('DOMContentLoaded', () => {
-        const paywall = document.getElementById('paywall-banner');
-        removeDOMElement(paywall);
+		const fence = document.querySelector('.fence-body');
+		if (fence)
+			fence.classList.remove('fence-body');
```

Comment:
To maintain readability you should use one of these examples:

```suggestion
		if (fence) fence.classList.remove('fence-body');
```

or

```suggestion
		if (fence) {
		    fence.classList.remove('fence-body');
		}
```